ballerina/build.gradle

```diff
@@ -166,7 +166,7 @@ task startBallerinaSTS() {
             if (!stdOut.toString().contains("ballerina-sts")) {
                 println "Starting Ballerina STS."
                 exec {
-                    commandLine 'sh', '-c', "docker run --rm -d -p 9445:9445 -p 9444:9444 --name ballerina-sts ldclakmal/ballerina-sts:latest"
+                    commandLine 'sh', '-c', "docker run --rm -d -p 9445:9445 -p 9444:9444 --name ballerina-sts sabthar/ballerina-sts:latest"
```

Comment:
Shall we add the github url of the docker image as well? Here or in the docker hub readme file.

Reply:
added docker hub readme link